modules/game/src/main/GameRepo.scala

```diff
@@ -196,7 +196,7 @@ final class GameRepo(val coll: Coll)(implicit ec: scala.concurrent.ExecutionCont
 
   // Use Env.round.proxy.urgentGames to get in-heap states!
   def urgentPovsUnsorted(user: User): Fu[List[Pov]] =
-    coll.list[Game](Query nowPlaying user.id, Game.maxPlayingRealtime) dmap {
+    coll.list[Game](Query nowPlaying user.id, Game.maxPlaying + 5) dmap {
```

Comment:
cant remember what this was for?

Reply:
it was to fix that : https://playstrategy.org/forum/playstrategy-feedback/does-not-appear

Reply:
or maybe you meant why we add +5 ?
https://github.com/Mind-Sports-Games/lila/pull/915#discussion_r1751591781